### ISSUE TITLE
Recruit a friend

### ISSUE THREAD

**Issue by Dynamit123** [dalaranwow/dalaran-wow#5588]
Recruit a friend
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)

**Expected behaviour**: Tell us what should happen instead

**Steps to reproduce the problem**:

1. Recruit a friend
2. Should send email to friend but it does not
3. 

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Golganeth** [comment]
This is the in-game bugtracker, which lists in-game mechanic issues that can affect players.
Make sure you followed all the steps explained on the forums: **[>click me<](http://www.dalaran-wow.com/forums/news-and-rules/rules-and-guidelines/topic/2565/how-to-use-the-recruit-a-friend).**
Also please make sure to check that the RaF invitation isn't in your **[spam folder](http://www.dalaran-wow.com/forums/input-and-assistance/community-support/topic/11799/raf-invites-sent-but-not-received).**

If the issue persists, please contact us at **dalaran.community@gmail.com** explaining your problem. Include your account name(s), e-mail(s) and any other relevant info.

Closing up
